src/utils/registerPlugin.ts

```diff
@@ -0,0 +1,19 @@
+import { EventResponse } from '@fingerprintjs/fingerprintjs-pro-server-api'
```

Comment:
Is it possible to move this file under `plugins`? The reason is that, the `registerPlugins` is used by plugins, and if we decide to change its path in future, it would be a breaking change (import paths would be broken). I guess, in JS world we could re-export it in the previous path to avoid breaking changes. What do you think @Orkuncakilkaya is the best strategy here?

Reply:
Keeping it inside the utils folder made more sense because I expect plugins folders to contain only plugins.

Reply:
so you agree that, in future if we were to move this file to some other location, you like to re-export it to old location? Why is this better than moving this function to `plugins` folder?

Reply:
The plugins folder is where customers keep their own work, so if we put a file there, it can be confusing for them. Also, we expect each file inside the plugins folder to register a plugin. I believe we will not move this register plugin function to another location in the future, because it's a utility function. If you don't think it's utility function what do you think it's. I think we should either move it now or never move it